### PR TITLE
setup project structure

### PR DESCRIPTION
**Pull request by Max2306M**

### CODE REVIEW

**Review by TetianaUp — APPROVED** [review]

File: src/test/java/base/AbstractBasePage.java [hunk 1]
```diff
@@ -3,33 +3,36 @@
...
 import java.util.List;
 
+import static java.time.Duration.ofMillis;
+import static org.openqa.selenium.support.ui.ExpectedConditions.*;
+
 abstract public class AbstractBasePage {
     protected WebDriver driver;
     protected WebDriverWait wait;
-    protected Actions actions;
+    private int BASE_WAIT = 5000;
```
Comment (SkipperV) [inline src/test/java/base/AbstractBasePage.java:16]: private final int.. 
as it is a constant value

File: src/test/java/pages/CarsInHeaderPage.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class CarsInHeaderPage extends AbstractBasePage {
+
+    public CarsInHeaderPage(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final String CARS_IN_HEADER = "//a[@aria-label='Автомобілі']";
```
Comment (SkipperV) [inline src/test/java/pages/CarsInHeaderPage.java:13]: Put this static and getter into page elements class

File: src/test/java/pages/HomePage.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends AbstractBasePage {
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final String CARS_IN_AVAILABILITY = "//a[@id='button-f4e1544f40'] ";
```
Comment (SkipperV) [inline src/test/java/pages/HomePage.java:13]: Put this static and getter into page elements class

File: src/test/java/tests/TestCarsInHeader.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+import base.AbstractBaseTest;
+import org.testng.annotations.Test;
+import pages.HomePage;
+
+public class TestCarsInHeader extends AbstractBaseTest {
+
+    @Test
+    public void test() {
+        openPage("https://www.bmwusa.com/");
```
Comment (SkipperV) [inline src/test/java/tests/TestCarsInHeader.java:11]: Incorrect site.
https://www.bmw.ua/uk/index.html

**Review by SkipperV — CHANGES_REQUESTED** [review]
Some changes (written comments)

**Review by SkipperV — APPROVED** [review]
Approved.

**Pull request merged**